### ISSUE TITLE
用ERINE训练自己的NER数据集时，F1和precision都是0

### ISSUE THREAD

**Issue by SuMarsss** [PaddlePaddle/ERNIE#183]
跑测试的[msra_ner](https://ernie.bj.bcebos.com/task_data.tgz)可以跑通，f1能达到95，但是当我用ERINE跑自己的NER数据时，f1和precision都是0
测试结果：
> epoch: 6, progress: 5248/6144, step: 2580, loss: 0.269858, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 3.507191 steps/s
> epoch: 6, progress: 5408/6144, step: 2590, loss: 0.245142, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 2.484499 steps/s
> epoch: 6, progress: 5568/6144, step: 2600, loss: 0.295873, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 2.440172 steps/s
> [dev evaluation] f1: 0.000000, precision: 0.000000, recall: 0.000000, elapsed time: 6.710833 s
> [test evaluation] f1: 0.000000, precision: 0.000000, recall: 0.000000, elapsed time: 6.444398 s
> epoch: 6, progress: 5728/6144, step: 2610, loss: 0.175324, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 0.579638 steps/s
> epoch: 6, progress: 5888/6144, step: 2620, loss: 0.284426, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 3.364358 steps/s
> epoch: 6, progress: 6048/6144, step: 2630, loss: 0.073127, f1: 0.000000, precision: 0.000000, recall: 0.000000, speed: 2.896042 steps/s

我已经按照教程里的ner数据格式整理过自己的数据集了，字符之间的特殊符号也教程里的'\x02'。
以下是我的tsv数据还有label_map.json.

> 胸部CT未见转移征象	B-RadiologyI-RadiologyI-RadiologyI-RadiologyOOOOOO

> {
>   "B-AnatomyPart": 0,
>   "I-AnatomyPart": 1,
>   "B-Diagnosis": 2,
>   "I-Diagnosis": 3,
>   "B-Drug": 4,
>   "I-Drug": 5,
>   "B-Lab": 6,
>   "I-Lab": 7,
>   "B-Procedure": 8,
>   "I-Procedure": 9,
>   "B-Radiology": 10,
>   "I-Radiology": 11,
>   "O": 12
> }

![image](https://user-images.githubusercontent.com/45394577/60415826-e3647e80-9c0e-11e9-9b15-acef8dfbe1b5.png)
![360截图18060115626665](https://user-images.githubusercontent.com/45394577/60415889-0c850f00-9c0f-11e9-8bc0-6333b7879aad.png)

![image](https://user-images.githubusercontent.com/45394577/60415836-ebbcb980-9c0e-11e9-8dd4-60ea1f649509.png)

**Comment by SuMarsss** [comment]
```
if steps % args.skip_steps != 0:
   train_exe.run(fetch_list=[])
```
还有源码run_sequence_labeling.py 中，为什么一直给train_exe.run传入空的fetch_list？

**Comment by tianxin1860** [comment]
空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；

**Comment by SuMarsss** [comment]
> 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；

那上面一个问题呢，为什么f1和precision都是0呢

**Comment by SuMarsss** [comment]
> 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；

怎么能检验下自己的NER数据格式对不对，有没有正确传入进去呢？有没有哪一个中间的变量的值，可以反映这个

**Comment by tianxin1860** [comment]
> > 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；
> 
> 那上面一个问题呢，为什么f1和precision都是0呢

可以在网络中把 infer 结果打印出来，检查一下哪个环节出错。

**Comment by SuMarsss** [comment]
> > > 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；
> > 
> > 
> > 那上面一个问题呢，为什么f1和precision都是0呢
> 
> 可以在网络中把 infer 结果打印出来，检查一下哪个环节出错。

哦哦，好的，现在已经解决f1，precision一直为0的问题了，但是训练结果的f1一直上不去，一直在20多左右，应该调一下模型中的哪些参数呢？learning_rate应该调大一些还是小一些呢

**Comment by tianxin1860** [comment]
> > > > 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；
> > > 
> > > 
> > > 那上面一个问题呢，为什么f1和precision都是0呢
> > 
> > 
> > 可以在网络中把 infer 结果打印出来，检查一下哪个环节出错。
> 
> 哦哦，好的，现在已经解决f1，precision一直为0的问题了，但是训练结果的f1一直上不去，一直在20多左右，应该调一下模型中的哪些参数呢？learning_rate应该调大一些还是小一些呢

1. f1, precision 一直为0是什么原因造成的？
2. 帖一下你当前的参数配置，以及验证集或测试集的各指标评估结果

**Comment by arbalest339** [comment]
> > > > 空的 fetch_list 是为了不影响训练速度，如果每个 batch 训练都去 fetch 的话会降低训练速度；
> > > 
> > > 
> > > 那上面一个问题呢，为什么f1和precision都是0呢
> > 
> > 
> > 可以在网络中把 infer 结果打印出来，检查一下哪个环节出错。
> 
> 哦哦，好的，现在已经解决f1，precision一直为0的问题了，但是训练结果的f1一直上不去，一直在20多左右，应该调一下模型中的哪些参数呢？learning_rate应该调大一些还是小一些呢

我也想请问您f1，precision一直为0的问题是怎么解决的，我用原始的数据和参数跑NER任务，一开始也总是f1,precision为0，后面虽然不为0了但是也一直上不去，从loss值上看感觉像是收敛了，但是在验证集和测试集上的表现一直很差劲，三项指标一直不过0.5。
